### DISCUSSION TITLE
Rectangle Pro w/ Sonoma 14.3 All Menu Options Grayed Out

### DISCUSSION THREAD

**Discussion by efaden** [rxhanson/RectanglePro-Community#622] [Q&A]
Fresh install of rectangle pro.  The keyboard shortcuts seem to work, but when I go to click the menu bar icon literally every option is grayed out.  Anyone seen this?

**Comment by rxhanson** [comment]
This means the accessibility permissions didn't take. You'll need to remove and re-add them.

Manual instructions:
1. Quit Rectangle Pro
2. In System Settings -> Privacy & Security -> Accessibility, first disable Rectangle Pro, then remove it with the minus button. (it's important to do both of those steps in that order)
3. Launch Rectangle Pro and enable settings for it as prompted.

-or-

Terminal equivalent:
```
killall Rectangle\ Pro
tccutil reset All com.knollsoft.Hookshot
open -a Rectangle\ Pro
```
